benchmarks/mpas_ocean.py

```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
 
 import uxarray as ux
+from uxarray.grid.geometry import _construct_hole_edge_indices
```

Comment:
We need to add this import within the `HoleEdgeIndices` scope, since it's causing our other benchmarks to fail.

See the discussion in #811